experiments/local_linear_examples/bias_image.R

```diff
@@ -0,0 +1,43 @@
+set.seed(1234)
+
+rm(list = ls())
+
+setwd("~/git/grf/experiments/local_linear_examples")
```

Comment:
It's better not to hard code paths. In RStudio, you can do the following:
```
setwd(dirname(rstudioapi::getActiveDocumentContext()$path))
```

Reply:
I just saw that we hardcode the paths in some of the instrumental variables examples. This set a bad precedent, we can plan to clean those up.